roles/cri-o-install/tasks/install.yml

```diff
@@ -35,6 +35,7 @@
     - skopeo-containers
     - ostree
     - ostree-devel
+    - iptables
```

Comment:
Just `iptables`? I would have thought in Fedora you should be using `firewalld`

Reply:
firewalld will probably pull down iptables too, but, cri-o specifically complained about a raw iptables command it couldn't execute (unsure at which step, but it complained)